template/android/gradle.properties

```diff
@@ -42,3 +42,5 @@ newArchEnabled=false
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
 hermesEnabled=true
+
+VERSION_NAME=1.0.0
```

Comment:
why do you add `VERSION_NAME` to this line

Reply:
to create default value for `VERSION_NAME` if you build in local it will get value from here. if you build via pipeline it will take value from `fastlane`.

Reply:
okay thanks